__tests__/data/rockerbox_router_input.json

```diff
@@ -0,0 +1,168 @@
+            "eventName": ""
+          }
+        ],
+        "eventsMap": [
+          {
+            "from": "Product Added",
+            "to": "conv.add_to_cart"
+          }
+        ],
+        "useNativeSDK": {
+          "web": false
+        },
+        "clientAuthId": {
+          "web": "test-client-auth-id"
+        },
+        "oneTrustCookieCategories": {
+          "web": [
+            {
+              "oneTrustCookieCategory": "Marketing Sample"
+            }
+          ]
+        },
+        "customDomain": {
+          "web": "https://cookiedomain.com"
+        },
+        "enableCookieSync": {
+          "web": true
+        }
+      }
+    },
+    "metadata": {
+      "jobId": 1
+    },
+    "message": {
+      "type": "track",
+      "event": "Product Added",
+      "sentAt": "2022-08-07T20:02:19.352Z",
+      "userId": "userSampleX138",
+      "channel": "web",
+      "context": {
+        "os": {
+          "name": "",
+          "version": ""
+        },
+        "locale": "en-IN",
+        "traits": {
+          "email": "userSampleX120@gmail.com",
+          "phone": "9878764736",
+          "last_name": "Stack",
+          "first_name": "Rudder"
+        },
+        "campaign": {},
+        "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36"
+      },
+      "rudderId": "4a47e99b-2afc-45c6-b902-ed69282ca805",
+      "messageId": "1659902539347900-c622426c-a1dd-44c0-ac6d-d4dbee3f4a93",
+      "properties": {
+        "checkout_id": "12345",
+        "product_url": "http://www.yourdomain.com/products/red-t-shirt",
+        "product_name": "Red T-shirt"
+      },
+      "anonymousId": "5f093403-1457-4a2c-b4e4-c61ec3bacf56",
+      "integrations": {
+        "All": true
+      },
+      "originalTimestamp": "2022-08-07T20:02:19.347Z"
+    },
+    "writeKey": "2D0yaayoBD7bp8uFomnBONdedcA",
+    "requestIP": "[::1]",
+    "receivedAt": "2022-08-08T01:32:19.369+05:30"
+  },
+  {
+    "destination": {
+      "Config": {
+        "advertiserId": "test id",
+        "eventFilteringOption": "disable",
+        "whitelistedEvents": [
+          {
+            "eventName": ""
+          }
+        ],
+        "blacklistedEvents": [
+          {
+            "eventName": ""
+          }
+        ],
+        "eventsMap": [
+          {
+            "from": "Product Viewed",
+            "to": "conv.add_to_cart"
+          }
+        ],
+        "useNativeSDK": {
+          "web": false
+        },
+        "clientAuthId": {
+          "web": "test-client-auth-id"
+        },
+        "oneTrustCookieCategories": {
+          "web": [
+            {
+              "oneTrustCookieCategory": "Marketing Sample"
+            }
+          ]
+        },
+        "customDomain": {
+          "web": "https://cookiedomain.com"
+        },
+        "enableCookieSync": {
+          "web": true
+        }
+      }
+    },
+    "metadata": {
+      "jobId": 1
```

Comment:
Why jobId is 1 here as well it should be 2

Reply:
Fixed.